docs/content/commands/manpages/raster/geowave-installgdal.txt

```diff
@@ -14,7 +14,7 @@ geowave-raster-installgdal - Install GDAL by downloading native libraries
 [[raster-installgdal-description]]
 ==== DESCRIPTION
 
-This command will download GDAL to a directory.  The directory should be added to the `PATH` and Linux users should set `LD_LIBRARY_PATH` to the directory.  This command is not currently supported on Mac.
+This command will download GDAL to a directory.  The directory should be added to the `PATH` and Linux users should set `LD_LIBRARY_PATH` to the directory.
```

Comment:
This description actually seems to be a bit out of date.  Setting the environment variables is only needed when overriding the download directory.

Also, in the `EXAMPLES` section, the text needs to be updated from `Install GDAL libraries on a Windows or Linux machine` to `Install GDAL native libraries`.